### PR TITLE
ABOS_SOTS: add new platform and remove obsolete code

### PR DESCRIPTION
**Pull request by mhidas**
* Add FluxPulse platform
* Disable IMOS checks for ABOS_SOTS
* Remove obsolete ABOS code.

### CODE REVIEW

File: watch.d/ABOS_SOTS.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "path": [ "ABOS/SOTS" ],
   "execute": "ABOS/common/incoming_handler.sh",
```
Comment (lbesnard) [inline watch.d/ABOS_SOTS.json]: I guess the handler you just removed
``` ABOS/SOTS/incoming_handler.sh``` hasn't be used for a long time anyway right ?

Comment (mhidas) [inline watch.d/ABOS_SOTS.json]: That's right. It's been replaced by the ABOS/common one.

**Comment by mhidas** [comment]
I've just pushed in another quick change to disable the IMOS checks for this pipeline. I have files coming in that are actually IMOS-1.4 compliant, but we don't have a checker for that yet!

**Comment by lbesnard** [comment]
@mhidas all good, ready to be merged ?

**Comment by mhidas** [comment]
@lbesnard Yes please!

**Pull request merged**